cypress/support/fragments/requests/requests.js

```diff
@@ -565,4 +565,22 @@ export default {
     body: requestBody,
     isDefaultSearchParamsRequired: false,
   }),
+
+  /* for miltiselect 'Pickup service point' we have to redefine attribute 'aria-labelledby' to make it unique,
+ because there are 4 elements with same 'aria-labelledby' on the page so the function 'createInteractor()'
+  in interactors\multi-select.js takes as argument the first one and it's not we needed
+  */
+  filterRequestsByServicePoints(servicePoint) {
+    // wait untill element is visible and interactable
+    cy.wait(3000);
+    cy.get('[id="req-pickup-service-point-filter"]').click();
```

Comment:
I think makes sense to make this element as a variable, because it is used twice here